kolibri/core/utils/csv.py

```diff
@@ -0,0 +1,8 @@
+import io
+import sys
+
+
+def open_csv_for_writing(filepath):
+    if sys.version_info[0] < 3:
+        return io.open(filepath, "wb")
+    return io.open(filepath, "w", newline="", encoding="utf-8-sig")
```

Comment:
If we are using utf-8-sig encoding by default when writing any csv file, we should do the same when reading it. Otherwise in non-windows system  we'll get a` \ufeff` char at the beginning of every file we open. For instance, if we use it in `bulkexportusers.py` to write the csv, we should use it in `bulkimportusers.py` when reading it.